float32[4] normalQuat
---
float32 distance